src/features/isanteplus/findPatient.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Patient Search
+
+  Background:
+    Given User logs in the system
+
+  @findPatient
+  Scenario: Searching for a patient
+  And User clicks on Find Patient App
+  And  User enters moses mutesa
+  Then Search Page returns patients
+  And  User clicks on first patient
+  Then System loads patient dashboard
```

Comment:
can you use    ` Selected patient’s `Cover Page` will be displayed.`

Reply:
Resolved thanks